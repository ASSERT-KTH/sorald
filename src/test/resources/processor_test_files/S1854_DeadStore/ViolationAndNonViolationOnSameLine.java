public class ViolationAndNonViolationOnSameLine
{
    public void dead()
    {
        int x = 5;
        int y = 2;
        System.out.println(x + y);
        x = 2; y = 3; // Noncompliant
        System.out.println(y);
    }
}
