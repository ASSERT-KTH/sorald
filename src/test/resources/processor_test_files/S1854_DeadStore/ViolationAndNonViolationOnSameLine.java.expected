public class ViolationAndNonViolationOnSameLine
{
    public void dead()
    {
        int x = 5;
        int y = 2;
        System.out.println(x + y);
        y = 3;
        System.out.println(y);
    }
}
